libs/input-mapping/src/Helper/FakeDevStorageTableRewriteHelper.php

```diff
@@ -52,14 +53,14 @@ private static function getNewSource(string $source, string $branchName): string
             $bucketId = substr($bucketId, 2);
             $prefix = 'c-';
         }
-        $bucketId = $branchName . '-' . $bucketId;
+        $bucketId = $branchId . '-' . $bucketId;
         $tableIdParts[1] = $prefix . $bucketId;
         return implode('.', $tableIdParts);
     }
 
-    private static function rewriteSource(string $source, ClientWrapper $clientWrapper, LoggerInterface $logger): string
+    private function rewriteSource(string $source, ClientWrapper $clientWrapper, LoggerInterface $logger): string
     {
-        $newSource = self::getNewSource(
+        $newSource = $this->getNewSource(
             $source,
             $clientWrapper->getBranchClientIfAvailable()->webalizeDisplayName(
```

Comment:
Co to je, ze webalizujeme `default` / ciselny id branche?

Reply:
jj, to je nějaká chujovina https://github.com/keboola/platform-libraries/pull/127#discussion_r1262393817